src/Sphere.cc

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *
 */
+#include <gz/math/Inertial.hh>
+
```

Comment:
```suggestion
#include <optional>
```

Reply:
Added the `<optional>` header to all source files where it was used in [8fef740](https://github.com/gazebosim/sdformat/pull/1299/commits/8fef7400ad028ca9015c8454e52e713a01f0129c)